docs/bundler/executables.md

```diff
@@ -45,6 +45,58 @@ When deploying to production, we recommend the following:
 bun build --compile --minify --sourcemap ./path/to/my/app.ts --outfile myapp
 ```
 
+### Cross-compile to other platforms
+
+To target a different platform, architecture, or version of Bun, use the `--target` flag.
+
+To build for Linux from macOS, Windows or Linux, you can use the following command:
+
+```sh
+# The default architecture is x64 if an os is specified.
```

Comment:
We should require the arch, this is going to cause people to be confused like with `--target=node`